docs/ApiGuidelines.md

```diff
@@ -0,0 +1,76 @@
+# API Guidelines
+
+## APIs
+
+There are three types of APIs: 
+
+1. Public
+2. Experimental
```

Comment:
Others argued during a meeting today that we don't need both Experimental and Preview.   That doesn't mean that question is closed, only that I'm not making any changes at this time since I saw no consensus on adding Preview as an api category.  We might consider it a separate issue to track though if you like.

Reply:
Trying to pair down the comments to make sure this is ok to merge. I'm closing this for now.